client-proxy/transports/http_transport.go

```diff
@@ -0,0 +1,10 @@
+package transports
+
+type HttpTransport struct {
+	OpenLineageURL    string
+	OpenLineageAPIKey string
+}
+
+func (transport *HttpTransport) Send(lineageEvent string) error {
```

Comment:
_Minor_: Similar comment as in https://github.com/OpenLineage/OpenLineage/pull/1420/files#r1054828563

Reply:
Here, `Emit` makes sense.